app/models/like.rb

```diff
@@ -1,4 +1,10 @@
 class Like < ApplicationRecord
-  belongs_to :user
-  belongs_to :post
+  belongs_to :user, class_name: 'User'
+  belongs_to :post, class_name: 'Post'
+
+  after_create :update_like_counter_post
+
+  def update_like_counter_post
+    post.update(likes_counter: post.likes.count)
+  end
```

Comment:
## Optional
- [ ] I suggest that you use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - after_save to call the `#update_like_counter_post` method every time a new like has been saved to the database, then make the `#update_like_counter_post` method private